src/components/Header/Header.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const Header = () => (
+  <header>
+      <nav>
+        <ul className="nav nav-tabs">
+          <li className="nav-item">
+            <Link className="nav-link" to="/">Home</Link>
```

Comment:
You could use `NavLink` with `activeClassName` attribute for highlighting current tab